lib/search.js

```diff
@@ -118,9 +123,16 @@ async function processFirstPage (html, opts, savedApps, mappings) {
   return checkFinished(opts, [...savedApps, ...apps], token);
 }
 
+function isMoreSection (section) {
+  const sectionTitle = R.path(SECTIONS_MAPPING.title, section);
+  return R.is(String, sectionTitle) && !R.isEmpty(sectionTitle);
```

Comment:
The first non-empty title does not guarantee it's the more_results section. Try a random query like `lajsdfkljew`. The first section should have a title: `No results for lajsdfkljew`. I think this case will break the script.

Reply:
This would return
```
{
    "message": "Keine Ergebnisse für <b>lajsdfkljew</b>"
}
```

Reply:
Okay never mind I'm kinda stupid😅. So yeah it still throws as it did before